app/javascript/components/patient/assessment/steps/AssessmentCompleted.js

```diff
@@ -12,44 +12,44 @@ class AssessmentCompleted extends React.Component {
   render() {
     return (
       <Card className="mx-0 card-square align-item-center">
-        <Card.Header className="text-center h4">{this.props.translations[this.props.lang]['web']['title']}</Card.Header>
+        <Card.Header className="text-center h4">{this.props.translations[`${this.props.lang}`]['web']['title']}</Card.Header>
```

Comment:
Curious why prettier is making this change? seems unnecessary

Reply:
this looks like a workaround for the generic object sink warning

Reply:
Exactly Tim.